tests/test_content_loader.py

```diff
@@ -1544,6 +1550,34 @@ def test_question_value_returns_matching_option_label(self):
 
         assert question.value == 'Option label'
 
+    def test_number_questions_without_unit(self):
+        question = ContentQuestion({
+            "id": "example",
+            "type": "number",
+        }).summary({'example': '12.20'})
+
+        assert question.value == '12.20'
+
+    def test_number_questions_adds_unit_before(self):
+        question = ContentQuestion({
+            "id": "example",
+            "type": "number",
+            "unit": u"£",
```

Comment:
I'm not sure if it would it be nicer to import `unicode_literals` for the whole test file?  Or better to keep it explicit?  Probably doesn't matter too much...